app/interactors/editions/create_interactor.rb

```diff
@@ -25,10 +25,23 @@ def find_and_lock_live_edition
     context.live_edition = edition
 
     assert_edition_state(edition, assertion: "can create new edition") do
-      edition.published? || edition.published_but_needs_2i? || edition.removed?
+      edition.published? ||
+        edition.published_but_needs_2i? ||
+        edition.removed? ||
+        edition.discarded?
```

Comment:
Not necessary for this PR but I wonder if we want to encapsulate this as a method on the edition since it's reached 4 states. Naming could be a pain though - best that comes immediately to mind is `supersedable?`

Reply:
As discussed, we decided we're happy for this to include withdrawn editions, so the check can be reduced to "live or discarded".